src/examples/plugins/CMakeLists.txt

```diff
@@ -61,3 +61,11 @@ endif()
 if(AIMRT_BUILD_WITH_PROTOBUF AND AIMRT_BUILD_GRPC_PLUGIN)
   add_subdirectory(grpc_plugin)
 endif()
+
+if(AIMRT_BUILD_PROXY_PLUGIN
+   AND AIMRT_BUILD_WITH_PROTOBUF
+   AND AIMRT_BUILD_WITH_ROS2
+   AND AIMRT_BUILD_WITH_ROS2_PLUGIN
+   AND AIMRT_BUILD_ZENOH_PLUGIN)
+  add_subdirectory(proxy_plugin)
+endif()
```

Comment:
Add Those dependency options into root CMakeLists.txt's cmake_dependent_options

Reply:
here is example , just need to add_subdirectory